Lib/pprint.py

```diff
@@ -236,6 +239,53 @@ def _pprint_ordered_dict(self, object, stream, indent, allowance, context, level
 
     _dispatch[_collections.OrderedDict.__repr__] = _pprint_ordered_dict
 
+    def _pprint_dict_view(self, object, stream, indent, allowance, context, level):
+        """Pretty print dict views (keys, values, items)."""
+        if isinstance(object, self._dict_items_view):
+            key = _safe_tuple
+        else:
+            key = _safe_key
+        write = stream.write
+        write(object.__class__.__name__ + '([')
+        if self._indent_per_level > 1:
+            write((self._indent_per_level - 1) * ' ')
+        length = len(object)
+        if length:
+            if self._sort_dicts:
+                entries = sorted(object, key=key)
+            else:
+                entries = object
+            self._format_items(entries, stream, indent, allowance + 1,
+                               context, level)
+        write('])')
+
+    def _pprint_abc_view(self, object, stream, indent, allowance, context, level):
```

Comment:
`abc` is too generic, please use something like `mapping_abc` or just `mapping` if it’s clear enough

Reply:
Thank you, renamed to `_pprint_mapping_abc_view`.